guides/common/modules/con_registering-hosts-by-using-global-registration.adoc

```diff
@@ -8,6 +8,9 @@ That gives you complete control over the host registration process.
 You can also customize the default templates if you need greater flexibility.
 For more information, see xref:Customizing_the_Registration_Templates_{context}[].
 
+If you use the Resource Quota plugin, registered hosts are automatically assigned to the `Unassigned` resource quota.
+For more information, see {AdministeringDocURL}limiting-host-resources[Limiting host resources] in _{AdministeringDocTitle}_.
```

Comment:
AFAIK, the plugin isn't supported in Satellite:

```suggestion
ifndef::satellite[]
If you use the Resource Quota plugin, registered hosts are automatically assigned to the `Unassigned` resource quota.
For more information, see {AdministeringDocURL}limiting-host-resources[Limiting host resources] in _{AdministeringDocTitle}_.
endif::[]
```

I've noticed that the assembly is [included in the Admin guide](https://github.com/theforeman/foreman-documentation/blob/master/guides/doc-Administering_Project/master.adoc?plain=1#L63) for the Satellite build but I'm not sure that's correct. Pls, check with Ewoud.

Reply:
I have created a PR to fix this for Satellite 6.17: https://github.com/theforeman/foreman-documentation/pull/3921

Reply:
I want to handle the issue on "master" in a separate PR.